### DISCUSSION TITLE
Mutiple animation on single click

### DISCUSSION THREAD

**Discussion by Arumugamsix** [google/model-viewer#2699] [Q&A]
I have a Car Model. for that I need to open all the doors with a single button click.

But only one door animation is working. The remaining does not work.

Animations are
 modelViewer.animationName = ['Boot_Door_Ani','Front_Door_LeftAction','Back_Door_Left_AniAction','Back_Door_Right_Ani','Front_Door_Right_AniAction'];

kindly give a solution

**Comment by elalish** [comment]
Add an animation to the file called "All_Doors" that combines those existing animations. Then select that name and play it.

**Reply by Intiserahmed** [reply]
@cdata  @elalish So it can not play a list of animation at once?

**Reply by elalish** [reply]
Correct. I'm not sure we want to expose the whole three.js animation mixer API given its complexity. It's an option, but not a high priority so far.

**Reply by Intiserahmed** [reply]
no easy way-around for this?

**Reply by elalish** [reply]
Easiest would be forking us and interfacing directly into the three.js animation system to do what you want to  do. Probably quite a bit easier is exporting a combined animation in your GLB.
